src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -967,6 +968,39 @@ class spell_dru_moonkin_form_passive : public AuraScript
     }
 };
 
+// 50464 - Nourish
+class spell_dru_nourish : public SpellScript
+{
+    PrepareSpellScript(spell_dru_nourish);
+
+    void HandleHeal(SpellEffIndex /*effIndex*/)
+    {
+        Unit* caster = GetCaster();
+
+        // Glyph of Nourish
+        if (AuraEffect const* aurEff = caster->GetAuraEffect(SPELL_DRUID_GLYPH_OF_NOURISH, EFFECT_0))
+        {
+            uint32 auraCount = 0;
+            int32 heal = GetHitHeal();
+
+            Unit::AuraEffectList const& periodicHeals = GetHitUnit()->GetAuraEffectsByType(SPELL_AURA_PERIODIC_HEAL);
+            for (AuraEffect const* hot : periodicHeals)
+            {
+                if (caster->GetGUID() == hot->GetCasterGUID())
```

Comment:
I know that in the original impl there was none, but, should we actually filter here for druid spells only?